pybamm/expression_tree/unary_operators.py

```diff
@@ -952,6 +952,9 @@ def _unary_new_copy(self, child):
         """See :meth:`UnaryOperator._unary_new_copy()`."""
         return boundary_value(child, self.side)
 
+    def _sympy_operator(self, child):
+        return child
```

Comment:
this should actually return an indication of the boundary value. The mathematical notation looks something like this:
<img width="46" alt="Screen Shot 2021-06-30 at 2 08 56 PM" src="https://user-images.githubusercontent.com/20817509/124010384-c05f3780-d9ac-11eb-9ceb-da158fd02bc5.png">
That's hard to replicate so we could instead do something like
```python3
if child.domain in ["negative particle", "positive particle"] and self.side == "right":
   return child + "^{surf}" # value on the surface of the particle
else:
   return child + f"^{self.side}"
```